test/hotspot/jtreg/compiler/sharedstubs/SharedStubToInterpTest.java

```diff
@@ -23,15 +23,21 @@
  */
 
 /**
- * @test SharedStubToInterpTest
- * @summary Checks that stubs to the interpreter can be shared for static or final method.
+ * @test id=C1
  * @bug 8280481
+ * @summary Checks that stubs to the interpreter can be shared for static or final method.
  * @library /test/lib
+ * @requires vm.opt.TieredStopAtLevel == null & vm.opt.TieredCompilation == null
+ * @requires os.arch=="amd64" | os.arch=="x86_64" | os.arch=="i386" | os.arch=="x86" | os.arch=="aarch64" | os.arch=="riscv64"
```

Comment:
Consider using `vm.simpleArch` in `@requires`:
```
@requires vm.simpleArch == "x86" | vm.simpleArch == "x64" | vm.simpleArch == "aarch64" | vm.simpleArch == "riscv64"
```

Reply:
Fixed. Thanks Vladimir!